src/Converter/Feature2Mesh.js

```diff
@@ -481,12 +495,59 @@ function featureToExtrudedPolygon(feature, options) {
     return new THREE.Mesh(geom, options.polygonMaterial);
 }
 
+/**
+ * Created Instanced object from on mesh
+ *
+ * @param {THREE.MESH} mesh Model 3D to instanciate
+ * @param {*} count number of instances to create (int)
+ * @param {*} ptsIn positions of instanced (array double)
+ * @returns {THREE.InstancedMesh} Instanced mesh
+ */
+function createInstancedMesh(mesh, count, ptsIn) {
+    var instancedMesh = new THREE.InstancedMesh(mesh.geometry, mesh.material, count);
+    var index = 0;
+    for (let i = 0; i < count * 3; i += 3) {
+        const mat = new THREE.Matrix4();
+        mat.setPosition(ptsIn[i], ptsIn[i + 1], ptsIn[i + 2]);
+        instancedMesh.setMatrixAt(index, mat);
+        index++;
+    }
+
+    instancedMesh.instanceMatrix.needsUpdate = true;
+
+    return instancedMesh;
+}
+
+/**
+ * Convert a [Feature]{@link Feature} to a Instanced 3d Model
+ *
+ * @param {Object} feature
+ * @returns {THREE.Mesh} mesh or GROUP of THREE.InstancedMesh
+ */
+function featureTo3DModel(feature) {
```

Comment:
it's more points/coordinates to  instanced meshes?

because we could imagine the same feature with a polygon